src/main/java/com/redhat/management/approval/Internal.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.management.approval;
+
+/**
+ * This class was automatically generated by the data modeler tool.
+ */
+
+public class Internal implements java.io.Serializable {
+
+    static final long serialVersionUID = 1L;
+
+    private java.lang.String org_id;
```

Comment:
Same as above, can we change variable to camel case?

Reply:
Same reason as above.